### PR TITLE
fix(integrations): fix emarsys wsse auth

### PR DESCRIPTION
**Pull request by hassan254-prog**
## Describe the problem and your solution
- At the time of implementing `SIGNATURE-WSSE` auth mode, we didn't have test credentials to test the implementation, this PR confirms that this is now tested and also fixes a small issue, also adding naming aliases for the respective fields. 
- Also fixed the validation to now include credentials within the `SIGNATURE` authmode.

## Testing instructions (skip if just adding/editing providers) 
- When creating a new connection, Nango will throw an error if the credentials are wrong and successfully create one if credentials are correct, with the help of the`proxy.verification` field.

### CODE REVIEW

File: docs-v2/integrations/all/emarsys.mdx [hunk 1]
```diff
@@ -89,7 +89,7 @@ _No setup guide yet._
 
 - Emarsys offers different APIs(Emarsys core, Loyalty V4 and SMS), currently Nango support Emarsys core API which offers WSSE over SSL authentication.
 - When creating a new connection in Nango, use a combination of the `username` and `secret` that you received during onboarding. The `username` follows the format `account_name00X`, where X is a digit.
-
+- The Oauth Token has a fetch rate limiting of per `client_id`: 2 requests per minute, 10 requests per hour or per `customer_id`: 10,000 requests per day. For more details [OAuth Token Fetch Rate Limiting](https://dev.emarsys.com/docs/emarsys-core-api-guides/branches/OAuth-2.0/b3c3a1eba8515-authentication-in-v3-api#oauth-token-fetch-rate-limiting)
```
Comment (nalanj) [inline docs-v2/integrations/all/emarsys.mdx]:
```suggestion
- The OAuth Token has a fetch rate limiting of per `client_id`: 2 requests per minute, 10 requests per hour or per `customer_id`: 10,000 requests per day. For more details [OAuth Token Fetch Rate Limiting](https://dev.emarsys.com/docs/emarsys-core-api-guides/branches/OAuth-2.0/b3c3a1eba8515-authentication-in-v3-api#oauth-token-fetch-rate-limiting)
```

**Review by nalanj — COMMENTED** [review]
Just one small nit. Code all looks good, but I was unable to test yet because it said someone else was logged into the account from 1P.

File: docs-v2/integrations/all/emarsys.mdx (hunk 1 above)
Comment (hassan254-prog) [reply docs-v2/integrations/all/emarsys.mdx]: I have updated this with the latest [commit](https://github.com/NangoHQ/nango/pull/3193/commits/09a7be168ddc42872ab38a7b7dd5a02f4d6b700f)

**Comment by hassan254-prog** [comment]
> Just one small nit. Code all looks good, but I was unable to test yet because it said someone else was logged into the account from 1P.

Could you please try now, I have logged out.

**Review by nalanj — APPROVED** [review]
Just tested the connection, worked like a charm 🚀

**Pull request merged**